gef-ui/server/utils/helpers.js

```diff
@@ -63,19 +64,15 @@ const mapSummaryList = (data, itemsToShow) => {
       return { html: '<span class="has-text-danger" data-cy="required">Required</span>' };
     }
 
-    if (val === null) {
+    /* Clean-Deep removes any properties with Null value from an Object. Therefore if all
+    properties are Null, this leaves us with an Empty Object. isEmpty checks to see if the
+    Object is empty or not. */
+    if (val === null || _isEmpty(cleanDeep(val))) {
```

Comment:
Could just have a nunjuck filter to display dash if there is no value?
```<p>{{ someValue | dashIfEmpty }}</p>```

https://github.com/foundry4/dtfs2/blob/master/gef-ui/server/nunjucks-configuration/filter-dashIfEmpty.js

I think either works though. Probably more of a personal preference. What do you think?

Reply:
The whole summary list is generated via JS, so there is html in the template.